src/components/cards/NavMenuCard/NavMenuCard.tsx

```diff
@@ -142,6 +142,29 @@ function NavMenuCard() {
                     />
                   );
               })}
+              <NavBarLink
+                path={''}
+                linkText={'Create'}
+                key={'create-ant-nav-button'}
+                onClick={
+                  walletAddress
+                    ? () =>
+                        dispatchGlobalState({
+                          type: 'setShowCreateAnt',
+                          payload: true,
+                        })
+                    : () => {
+                        dispatchGlobalState({
+                          type: 'setShowCreateAnt',
+                          payload: true,
+                        });
+                        dispatchGlobalState({
+                          type: 'setShowConnectWallet',
+                          payload: true,
+                        });
```

Comment:
do we have to worry about order here?

Reply:
Nope: this sets both the display of the CreateAntModal and ConnectModal to true, however the CreateAntModal has jsx that prevents it from rendering bassed on wallet connection. Once the wallet is connected the jsx allows the CreateAntModal to be rendered.